pkg/server/test/list_objects.go

```diff
@@ -31,15 +32,23 @@ func (x *mockStreamServer) Send(m *openfgav1.StreamedListObjectsResponse) error
 	return nil
 }
 
+func mustNewStruct(t *testing.T, v map[string]interface{}) *structpb.Struct {
```

Comment:
note self (@jon-whit ) - this should move up more generally to a test helper. It may be used in multiple places.